R/bmerge.R

```diff
@@ -89,8 +89,9 @@ bmerge = function(i, x, icols, xcols, roll, rollends, nomatch, mult, ops, verbos
           # common case of ad hoc user-typed integers missing L postfix joining to correct integer keys
           # we've always coerced to int and returned int, for convenience.
           if (verbose) cat("Coercing double column i.",names(i)[ic]," (which contains no fractions) to type integer to match type of x.",names(x)[xc],".\n",sep="")
-          val = as.integer(i[[ic]])
-          attributes(val) = attributes(i[[ic]])
+          val = i[[ic]]
```

Comment:
Does this affect `i` in user's environment? I think it shouldn't but check to be sure?

Reply:
Best to unit test to be sure